WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/ai/AboutProductSubScreen.kt

```diff
@@ -41,93 +59,194 @@ fun AboutProductSubScreen(viewModel: AboutProductSubViewModel, modifier: Modifie
             state,
             viewModel::onProductFeaturesUpdated,
             viewModel::onDoneClick,
-            viewModel::onChangeToneClicked,
+            viewModel::onNewToneSelected,
             modifier
         )
     }
 }
 
+@OptIn(ExperimentalMaterialApi::class)
 @Composable
 fun AboutProductSubScreen(
     state: UiState,
     onProductFeaturesUpdated: (String) -> Unit,
     onCreateProductDetails: () -> Unit,
-    onChangeTone: () -> Unit,
+    onToneSelected: (AiTone) -> Unit,
     modifier: Modifier = Modifier
 ) {
-    var isFocused by remember { mutableStateOf(false) }
-    Column(
-        modifier = modifier
-            .background(MaterialTheme.colors.surface)
-            .padding(
-                start = dimensionResource(id = R.dimen.major_100),
-                end = dimensionResource(id = R.dimen.major_100),
-                top = dimensionResource(id = R.dimen.major_200)
+    val coroutineScope = rememberCoroutineScope()
+    val modalSheetState = rememberModalBottomSheetState(
+        initialValue = Hidden,
+        confirmStateChange = { it != HalfExpanded },
+    )
+    val configuration = LocalConfiguration.current
+
+    ModalBottomSheetLayout(
+        sheetState = modalSheetState,
+        sheetShape = RoundedCornerShape(topStart = 12.dp, topEnd = 12.dp),
```

Comment:
I think the value should be 8.dp, or better `dimenResource(id = R.dimen.minor_100)`
```suggestion
        sheetShape = RoundedCornerShape(
            topStart = dimensionResource(id = R.dimen.minor_100),
            topEnd = dimensionResource(id = R.dimen.minor_100)
        ),
```